bin/AMR_process.py

```diff
@@ -140,7 +140,7 @@ def process_AMR(blast_output_tsv: str, amr_finder_output_tsv:str, catalogue_file
     
     if not Path(amr_finder_output_tsv).is_file():
         logging.error("{} is not a file.".format(amr_finder_output_tsv))
-        raise FileNotFoundError
+        # raise FileNotFoundError  # 
```

Comment:
Any reason to undo a sensible Exception?

Reply:
By raising the error it would stop the whole script. I had some samples were amrfinder didn't have any output file, but blast did. It seemed better to just give a summary based on blast rather than stop.

Reply:
But maybe more sensible just to give the error and stop